internal/pkg/coordinator/monitor.go

```diff
@@ -204,6 +209,7 @@ func (m *monitorT) ensureLeadership(ctx context.Context) error {
 			m.log.Debug().Str("index", m.policiesIndex).Msg(es.ErrIndexNotFound.Error())
 			return nil
 		}
+		m.log.Debug().Err(err).Msg("Encountered error while querying policies")
```

Comment:
Same here as above, it will be logged anyways so you can skip the debug log message and enahnce the error itself.